FixMyBerlin/fixmy.frontend
# Setup react-intl

Add i18n capabilities via react-intl / format.js

Part of https://github.com/FixMyBerlin/fixmy.platform/issues/397

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Visit research page
- Observe language being set based on browser settings
- Switch language using buttons below table of contents

## Checklist:
